lib/msal-browser/docs/iframe-usage.md

```diff
@@ -0,0 +1,141 @@
+
+Additionally, when 3rd party cookies are disabled in **Chrome**, iframed MSAL apps will not have access to local or session storage. MSAL will fallback to in-memory storage in this case.
+
+## Single sign-on
+
+You **can** achieve [single sign-on](https://docs.microsoft.com/azure/active-directory/develop/msal-js-sso) between iframed and parent apps with the [same-origin](https://developer.mozilla.org/docs/Web/Security/Same-origin_policy) **and** with [cross-origin](https://developer.mozilla.org/docs/Web/Security/Same-origin_policy#cross-origin_script_api_access) **if** you pass an [account hint](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/login-user.md#silent-login-with-ssosilent) from the parent app to the iframed app.
+
+### Apps with same-origin
+
+Iframed and parent apps with the same-origin may have access to the same MSAL.js cache instance and be able to sign-in without prompts, provided that both apps configure MSAL to use the [local storage](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/caching.md#cache-storage) for caching. See for more: [Single sign-on with MSAL.js](https://docs.microsoft.com/azure/active-directory/develop/msal-js-sso)
+
+### Apps with cross-origin
+
+Iframed and parent apps with cross-origin can make use of the [ssoSilent()](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/login-user.md#silent-login-with-ssosilent) API to achieve single sign-on. To do so, the parent app needs to pass down either an **account**, a **loginHint** (username) or a **session id** (sid) to the iframed app. For cross-origin communication between iframed and parent apps, there are a few alternatives you can consider:
+
+- You can add query strings to iframe's source in the parent app and retrieve them later in the child:
+
+```javascript
+// Create the main myMSALObj instance
+// configuration parameters are located at authConfig.js
+const myMSALObj = new msal.PublicClientApplication({
+    auth: {
+        clientId: "ENTER_CLIENT_ID",
+        authority: "https://login.microsoftonline.com/ENTER_TENANT_ID",
+        redirectUri: "/redirect", // set to a blank page for handling auth code response via popups
+    },
+    cache: {
+        cacheLocation: "localStorage", // set your cache location to local storage
+    },
+});
+
+let username = "";
+
+window.onload = () => {
+    
+    const urlParams = new URLSearchParams(window.location.search);
+    username = urlParams.get("username");
+
+    // attempt SSO
+    myMSALObj.ssoSilent({
+        loginHint: username
```

Comment:
Can we have these samples demonstrate using `sid` or the account instead? `loginHint` causes a lot of issues for people and I think our docs should default to showing the more reliable hint when possible

Reply:
ah, ok then I'll revise

Reply:
done